packages/kbn-server-route-repository/src/route_validation_object.ts

```diff
@@ -14,7 +14,7 @@ export const routeValidationObject = {
   // if any validation is defined. Not having validation currently
   // means we don't get the payload. See
   // https://github.com/elastic/kibana/issues/50179
-  body: schema.nullable(anyObject),
+  body: schema.nullable(schema.oneOf([anyObject, schema.string()])),
```

Comment:
Adding `schema.string()` as an allowed type to accept plain text request bodies. 

This abstraction implements its own validation logic. Existing endpoints are not affected or compromised by this change since this snippet is essentially bypassing the platform schema validation tooling.

Reply:
Is this one example of where the API might accept YAML as the body?

Reply:
where is the body parsed if it's actually JSON? does this happen in Core?